pipeline/notifications/render_check_message.py

```diff
@@ -0,0 +1,28 @@
+from pipeline.notifications.renderer import RenderNotification
+from pipeline.types import NotificationInfo
+
+
+class NewMessageRender(RenderNotification):
+    def __init__(self):
+        pass
+
+    def render(self, inp: NotificationInfo) -> str:
+
+        # if the notification type doesn't match the type of render, return immediately saying as much
+        if inp.notification_type.value != "check_message":
+            return "Renderer does not support this type of notification."
+        str_msg_lst = ""
+        if len(inp.messagers) == 2:
+            str_msg_lst = " and ".join(inp.messagers) + "."
+        elif len(inp.messagers) > 2:
+            x = 0
+            for msg in inp.messagers:
+                if x == len(inp.messagers) - 1:
+                    str_msg_lst += f"and {msg}."
+                    break
+                str_msg_lst += f"{msg}, "
+                x += 1
+        else:
+            str_msg_lst = "".join(inp.messagers) + "."
```

Comment:
You can move this into a helper function so that (a) it could be reused and (b) the renderer code is more readable. Then it would look like this:

```python
messagers_list = get_messagers_names(inp.messager)
text = f"..."
return text
```

You can put the helper function in this file for now since it is currently only relevant to this use case. If others have a need for it in the future, they can refactor it by moving it to common code.